src/TeleportJoin.sol

```diff
@@ -199,55 +199,55 @@ contract WormholeJoin {
         }
         totalFee = fee + operatorFee;
         postFeeAmount = amtToTake - totalFee;
-        daiJoin.exit(bytes32ToAddress(wormholeGUID.receiver), postFeeAmount);
+        daiJoin.exit(bytes32ToAddress(teleportGUID.receiver), postFeeAmount);
 
         if (fee > 0) {
             vat.move(address(this), vow, fee * RAY);
         }
         if (operatorFee > 0) {
-            daiJoin.exit(bytes32ToAddress(wormholeGUID.operator), operatorFee);
+            daiJoin.exit(bytes32ToAddress(teleportGUID.operator), operatorFee);
         }
 
-        emit Mint(hashGUID, wormholeGUID, amtToTake, maxFeePercentage, operatorFee, msg.sender);
+        emit Mint(hashGUID, teleportGUID, amtToTake, maxFeePercentage, operatorFee, msg.sender);
     }
 
     /**
     * @dev External authed function that registers the wormwhole and executes the mint after
```

Comment:
` ... registers the "wormwhole"...` -> ` ... registers the teleport...`

The rest LGTM

Reply:
Fixed!